From 3fbfa72ed906fb6b5bb0350a37c71176fab7f452 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@gmail.com>
Date: Fri, 10 Apr 2026 14:47:07 +0000
Subject: [PATCH] format_attribute.h: fix typo in macro name
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes:

libpbnjson-native/2.15.0-22/git/include/public/pbnjson/c/compiler/format_attribute.h:17: warning: header guard ‘FORMAT_ATTRIBUTE_H_’ followed by ‘#define’ of a different macro [-Wheader-guard]
   17 | #ifndef FORMAT_ATTRIBUTE_H_
libpbnjson-native/2.15.0-22/git/include/public/pbnjson/c/compiler/format_attribute.h:18: note: ‘FORMAT_ATTIRBUTE_H_’ is defined here; did you mean ‘FORMAT_ATTRIBUTE_H_’?
   18 | #define FORMAT_ATTIRBUTE_H_

Upstream-Status: Pending [webOS OSE is no longer maintained on github]
Signed-off-by: Martin Jansa <martin.jansa@gmail.com>
---
 include/public/pbnjson/c/compiler/format_attribute.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/public/pbnjson/c/compiler/format_attribute.h b/include/public/pbnjson/c/compiler/format_attribute.h
index 7cb6557..72f76e1 100644
--- a/include/public/pbnjson/c/compiler/format_attribute.h
+++ b/include/public/pbnjson/c/compiler/format_attribute.h
@@ -15,7 +15,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #ifndef FORMAT_ATTRIBUTE_H_
-#define FORMAT_ATTIRBUTE_H_
+#define FORMAT_ATTRIBUTE_H_
 
 #if __GNUC__
 #include "gcc/format_attribute.h"
@@ -32,4 +32,4 @@
 
 #endif
 
-#endif /* FORMAT_ATTIBUTE_H_*/
+#endif /* FORMAT_ATTRIBUTE_H_ */
